docs/system/development.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Software Development
+tags:
+  - HMX
+  - MX-4
+  - CT
+  - MX-V
+  - T30
+  - T30 FR
+  - C61
+  - Development
+---
+
+## Overview
+
+Host Mobility Hardware run a Linux based system. As such, software is usually built from source. The target architechure is ARM32 and ARM64 which is different from typical desktops and servers so a cross-compiler is needed.
```

Comment:
```suggestion
Host Mobility computers run GNU/Linux where almost all software is built from source. The target architectures are ARM32 and ARM64. Since the typical development machines are x86_64, a cross-compiler is usually required.
```